app/views/my/notifications/_notification.html.haml

```diff
@@ -0,0 +1,7 @@
+.notification
```

Comment:
Could you change this line to `.notification{class: ("read" if notification.read?)}` please